synapse/models/proj.py

```diff
@@ -47,11 +38,63 @@ def getModelDefs(self):
 
             ('proj', {
 
+                'interfaces': (
+                    ('proj:task', {
+
+                        'doc': 'A common interface for tasks.',
+
+                        'template': {
+                            'task': 'task'},
+
+                        'props': (
+
+                            ('id', ('str', {'strip': True}), {
+                                'doc': 'The ID of the {task}.'}),
+
+                            ('project', ('proj:project', {}), {
+                                'doc': 'The project containing the {task}.'}),
+
+                            ('status', ('int', {}), {
+                                # TODO: make runtime setable int enum typeopts
+                                'doc': 'The status of the {task}.'}),
+
+                            ('priority', ('meta:priority', {}), {
+                                'doc': 'The priority of the {task}.'}),
+
+                            ('created', ('time', {}), {
+                                'doc': 'The time the {task} was created.'}),
+
+                            ('updated', ('time', {}), {
```

Comment:
Do we want to keep `ismax` on the updated prop?

Reply:
The specific prop could override the behavior either way so I think its mostly just what we want the default to be. Leaving it off seems like a reasonable default to me.